module1/build tools/multi-project/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'java'
+}
+
+group 'com.epam'
+version '1.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    compile files('C:\\Users\\kirill\\Desktop\\lab\\module1\\build tools\\jar\\core.jar')
```

Comment:
According to the task, multi-project consists of 2 sub-projects, but not depends on them.

Reply:
Added modules api and core to multi-project.